src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/FindSmallIndelRegions.java

```diff
@@ -99,9 +229,18 @@ protected void runTool( final JavaSparkContext ctx ) {
         private int curEnd;
 
         public Finder( final ReadMetadata readMetadata, final SVReadFilter filter ) {
+            this(readMetadata, filter, new SVIntervalTree<>(), null);
+        }
+
+        public Finder( final ReadMetadata readMetadata,
+                       final SVReadFilter filter,
+                       final SVIntervalTree<SVTypeLen> trueIntervals,
+                       final String cdfDir ) {
             this.readMetadata = readMetadata;
             this.filter = filter;
-            final Map<String, FragmentLengthStatistics> libraryStatisticsMap = readMetadata.getAllLibraryStatistics();
+            this.trueIntervals = trueIntervals;
+            this.cdfDir = cdfDir;
+            final Map<String, LibraryStatistics> libraryStatisticsMap = readMetadata.getAllLibraryStatistics();
             libraryToHistoPairMap = new HashMap<>(SVUtils.hashMapCapacity(libraryStatisticsMap.size()));
             libraryStatisticsMap.forEach( (libName, stats) -> {
                 final IntHistogram[] pair = new IntHistogram[2];
```

Comment:
Since the array is always size 2 (for the two halves), using a `Tuple2` is probably a better choice; but your call.